### DISCUSSION TITLE
Can't upload identitiy document as students

### DISCUSSION THREAD

**Discussion by yusupabdulsahru** [community/community#30299] [GitHub Education]
hello, there's no option for upload document on github student dev pack? available option is take pict

**Comment by Santosh-Prasad-Verma** [comment]
Hi! 👋
This is expected. GitHub Student Developer Pack doesn’t allow document uploads anymore — they only support “Take a photo” using your camera.

GitHub does this for verification reasons, so the document is captured live and can’t be reused or edited. That’s why you won’t see any “upload file” option.

Just take a clear photo of a valid student proof like:

College ID card

Bonafide certificate

Fee receipt with your name and college name

Make sure the text is readable and the photo isn’t blurry.
That’s all you need — nothing is wrong on your side.
